main.py

```diff
@@ -110,6 +119,8 @@ async def process_birthday(message, state):
         async with state.proxy() as data:
             data['birthday'] = birthday
             data['congratulations'] = False
+            user = User(data['name'], data['birthday'], message.chat.id)
+            data_manager.add(user)
     await Form.next()
     await message.reply('Введи любое сообщение что бы начать!')
 
```

Comment:
It's might better use another naming for the file couse we have full bot code with same filename. it's an example code, I guess, so you can use 'example.py' naming. Also you can bring code to a script format to demonstrate how it's work.